coderd/rbac/roles.go

```diff
@@ -195,6 +196,8 @@ func ReloadBuiltinRoles(opts *RoleOptions) {
 			ResourceGroup.Type:        {ActionRead},
 			// Org roles are not really used yet, so grant the perm at the site level.
 			ResourceOrganizationMember.Type: {ActionRead},
+			// Template admins can read all template insights data
```

Comment:
nit: template admins and auditors?

Reply:
The context here is for the template admin role, so I think the comment is still valid.

Reply:
Sorry, didn't notice that. Thanks for clarifying.